packages/model/src/gp2/working-group.ts

```diff
@@ -1,4 +1,5 @@
 import { ListResponse } from '../common';
+import { Resource } from './common';
```

Comment:
To simplify, you could put:
import { ListResponse, Resource } from '../common';